src/app.js

```diff
@@ -96,7 +102,7 @@ app.use(setContentId);
 app.use(checkSessionAndRedirect);
 
 app.get("/welcome", (req, res) => {
-  res.render("home.njk");
+  res.render("home.njk", { req: req });
```

Comment:
Why have you add the entire request variable here?

Reply:
Oh sorry, this was from me experimenting with another approach. I'll go ahead and remove it.